tests/system/node/5/config.yaml

```diff
@@ -14,14 +14,12 @@ node:
   timeout: 100
   # Path to the data directory (if the path doesn't exist it will be created)
   data_dir: .cache
-
-  tx_payload_max_size: 1024
```

Comment:
I think the `tx_payload_max_size` is missing. Is it intended?

Reply:
Yes. It was set to its default value.